### DISCUSSION TITLE
Add username/password with AEM / managed configuration

### DISCUSSION THREAD

**Discussion by eingemaischt** [schwabe/ics-openvpn#1531] [General]
I am using TinyMDM to deploy your openvpn client to different Androids.

Is it possible to supply Username & Password in an openvpn profile with the "Managed Configuration" feature?

**Comment by schwabe** [comment]
You can embed username and password in the config:

```
<auth-user-pass>
user
pass
</auth-user-pass>
```

**Comment by eingemaischt** [comment]
Hi Arne! Thank you very much for this speed reply!
